module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,18 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& list){
+    std::deque<std::string> queue;
+    for(auto& el : list){
+        queue.push_back(el);
+    }
```

Comment:
You don't modify element here so you should use const &
```suggestion
    for(const auto& el : list){
        queue.push_back(el);
    }
```